easybuild/easyblocks/f/fftw.py

```diff
@@ -187,9 +188,14 @@ def run_all_steps(self, *args, **kwargs):
                     if prec == 'single' and getattr(self, flag):
                         prec_configopts.append('--enable-%s' % flag)
 
-                # NEON (ARM) only for single precision and double precision (on AARCH64)
-                if (prec == 'single' and (self.asimd or self.neon)) or (prec == 'double' and self.asimd):
-                    prec_configopts.append('--enable-neon')
+                if self.sve:
+                    # SVE (ARM) only for single precision and double precision (on AARCH64 if sve feature is present)
+                    if prec == 'single' or prec == 'double':
+                        prec_configopts.append('--enable-sve --enable-armv8-cntvct-el0')
+                elif self.asimd or self.neon:
+                    # NEON (ARM) only for single precision and double precision (on AARCH64)
+                    if prec == 'single' or prec == 'double':
```

Comment:
`double` should only be enabled if `asimd` is supported?

```python
if prec == 'single' or (prec == 'double' and self.asimd`):

Reply:
fixed via force push